no-computer.md

```diff
@@ -46,6 +77,9 @@ aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+Not sure.  I'd argue that there are a number of different ways of depicting the
+reaction, however I'd say in general the format/syntax isn't substantially 
+different.  More like different dialects, not different languages.
```

Comment:
You're using a whole different set of words when you depict the reaction in a different way, though! That feels different enough that it wouldn't qualify as a dialect.